### ISSUE TITLE
Segmentation Fault in --derep_fulllength when --uc is used

### ISSUE THREAD

**Issue by your-highness** [torognes/vsearch#333]
Dear VSEARCH devs,

I installed VSEARCH 2.8.2 v2.8.2_linux_x86_64 via `conda install vsearch` from bioconda.

When using the example pipeline on MiSeq_SOP data, vsearch reports a Segmentation Fault when executing the command
```bash
    $VSEARCH --threads $THREADS \
        --derep_fulllength $s.filtered.fasta \
        --strand plus \
        --output $s.derep.fasta \
        --sizeout \
        --uc $s.derep.uc \
        --relabel $s. \
        --fasta_width 0
```

Thus, the file `$s.derep.fasta` is empty. If I leave out the parameter `--uc $s.derep.uc`, the dereplication finishes without errors.

I get the same error if I clone the repository and build it.

Do you have any idea how to solve that issue?

Thanks in advance!

**Comment by torognes** [comment]
Thank you for reporting this bug. I have comfirmed it and will look into it now.

**Comment by torognes** [comment]
The bug is fixed in version 2.8.3 just released.
